raft/src/main/java/org/apache/kafka/raft/KafkaRaftClient.java

```diff
@@ -740,22 +804,33 @@ private void handleResponse(RaftResponse.Inbound response) throws IOException {
     }
 
     private void handleRequest(RaftRequest.Inbound request) throws IOException {
-        ApiMessage requestData = request.data();
+        ApiKeys apiKey = ApiKeys.forId(request.data.apiKey());
```

Comment:
This is for the old line 726 `validateLeaderOnlyRequest`: it should be `validateLeaderOnlyResponse`.

Reply:
I think one of the methods should be `validateVoterOnlyRequest` and the other should be `validateLeaderOnlyRequest`. I made this change. Was there another invalid naming?

Reply:
That's good enough.